Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -2,7 +2,34 @@ import MapboxNavigationNative
 import MapboxDirections
 @_implementationOnly import MapboxCommon_Private
 
-class Navigator {
+protocol NavigatorProtocol {
```

Comment:
Naming is hard for me, but I think we could use something better for this one. `Protocol` suffix could be replaced with something what describes what such object actually does.